### PR TITLE
Robert/1132 again

### PR DESCRIPTION
**Pull request by robert-e-davidson3**

### CODE REVIEW

File: packages/augur-state/src/db/DB.ts
Comment (robert-e-davidson3) [inline packages/augur-state/src/db/DB.ts:120]: `this.trackedUsers.getUsers()` can fail. If it does then the promises built from `syncableDatabases` will be unhandled. Also somehow this caused a subsequent call to `DB.sync` to fail.

File: packages/augur-core/source/libraries/EthersFastSubmitWallet.ts
Comment (robert-e-davidson3) [inline packages/augur-core/source/libraries/EthersFastSubmitWallet.ts]:
```suggestion
```

File: azure-pipelines.yml
Comment (adrake33) [inline azure-pipelines.yml:115]: What's the issue with this test?

File: packages/augur-core/source/support/Dockerfile
Comment (adrake33) [inline packages/augur-core/source/support/Dockerfile]: Should this be removed?

File: packages/augur-state/src/db/DB.test.ts
Comment (adrake33) [inline packages/augur-state/src/db/DB.test.ts]: Nit: this line can be combined with the other @augurproject/artifacts import above.

File: packages/augur-state/src/db/DB.ts
Comment (adrake33) [reply packages/augur-state/src/db/DB.ts:120]: Good find! Thanks for fixing this.

File: packages/augur-state/src/db/Rollback.test.ts
Comment (adrake33) [inline packages/augur-state/src/db/Rollback.test.ts:10]: Seems like this should probably moved elsewhere so it can be reused across multiple tests instead of being duplicated.

File: packages/augur-state/src/db/test.ts
Comment (adrake33) [inline packages/augur-state/src/db/test.ts]: I moved this and userSpecificEvents to Augur.ts in https://github.com/AugurProject/augur/pull/1296. I also changed most of the userSpecificEvents to be genericEventNames after Paul explained that we'll actually want most of them to be considered generic. I think I should probably merge https://github.com/AugurProject/augur/pull/1296 first, and then you can update this file to import those from Augur.ts.

File: packages/augur-state/src/run.ts
Comment (adrake33) [inline packages/augur-state/src/run.ts]: This can also be combined with the import above.

File: packages/ethersjs-provider/src/EthersWeb3Provider.ts
Comment (adrake33) [inline packages/ethersjs-provider/src/EthersWeb3Provider.ts]: FYI, you should be able to use the existing EthersProvider instead once https://github.com/AugurProject/augur/pull/1296 gets merged.

**Review by adrake33 — APPROVED** [review]
Looks good overall! Just had a few minor suggestions and recommend waiting until https://github.com/AugurProject/augur/pull/1296 is merged before merging this.

File: azure-pipelines.yml
Comment (robert-e-davidson3) [reply azure-pipelines.yml:115]: @petong

File: packages/augur-core/source/support/Dockerfile
Comment (robert-e-davidson3) [reply packages/augur-core/source/support/Dockerfile]: @petong

File: packages/augur-state/src/db/DB.ts
Comment (robert-e-davidson3) [reply packages/augur-state/src/db/DB.ts:120]: :D

File: packages/augur-state/src/db/DB.test.ts
Comment (robert-e-davidson3) [reply packages/augur-state/src/db/DB.test.ts]: Oh yeah will change

File: packages/ethersjs-provider/src/EthersWeb3Provider.ts
Comment (robert-e-davidson3) [reply packages/ethersjs-provider/src/EthersWeb3Provider.ts]: Right! I'll do another PR to remove these after or maybe handle it in the merge if there's a conflict. Oh or just wait for #1296 to be merged.

File: azure-pipelines.yml
Comment (petong) [reply azure-pipelines.yml:115]: those tests are hard coded to the old directory structure.  I opened a ticket

https://github.com/AugurProject/augur/issues/1305

File: packages/augur-core/source/support/Dockerfile
Comment (petong) [reply packages/augur-core/source/support/Dockerfile]: yeah, we can remove it. I can either do it now or in another branch

Comment (robert-e-davidson3) [reply packages/augur-core/source/support/Dockerfile]:
```suggestion
```

File: packages/augur-state/src/db/test.ts
Comment (robert-e-davidson3) [reply packages/augur-state/src/db/test.ts]: KK

**Pull request merged**
